ui/src/app/app.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+export class AppComponent {  
+  title ='projekt-monsterz-front'
+  loginClick = false;
+  clickedOutsideVar = false;
+  formChosen = '';
+  clickOutside($event: any) {
+    if ($event.target.closest('app-login-screen') == null && this.loginClick == true && $event.srcElement && $event.srcElement.classList.contains('openmenu') == false) {
```

Comment:
From a style standpoint, it might be nice to separate this ... very, very long boolean check into a variable. Your call, though.